examples/tutorials/nb_python/replay_tutorial.py

```diff
@@ -81,6 +81,7 @@ def make_configuration():
     # Enable gfx replay save. See also our call to sim.gfx_replay_manager.save_keyframe()
     # below.
     backend_cfg.enable_gfx_replay_save = True
+    backend_cfg.create_magnum_renderer = make_video_during_simulation
```

Comment:
You haven't added a Python binding for the flag so I don't think this is going to do anything. Python doesn't complain when you assign to non-existent members (unfortunate in this case).

See https://github.com/facebookresearch/habitat-sim/pull/1258/files#diff-08c325c91ce6aa20a6124d796c72d68934251a35f1cbc6e24421e5c23617de69 .

Reply:
Hmmm, I still don't see a Python binding so I feel like this is still doing nothing. Can you add the binding and test that the flag is working with python scripts?

Reply:
We should add a Python unit test. You can test `sim.renderer is None` or similar.

Reply:
Oh wait, I forgot. This is just re-using the existing flag that already has a binding. Up to you if you want to bother with a Python unit test.